ui/src/app/cron-workflows/cron-workflow-list.tsx

```diff
@@ -222,8 +222,8 @@ export function CronWorkflowList({match, location, history}: RouteComponentProps
     );
 }
 
-function getCronNextScheduledTime(spec: CronWorkflowSpec): Date {
-    if (spec.schedule != '') {
+function getSpecNextScheduledTime(spec: CronWorkflowSpec): Date {
+    if (spec.schedule && spec.schedule !== '') {
```

Comment:
```suggestion
    if (spec.schedule) {
```

truthy covers the case of empty string already, so the `&& ..` is redundant